recipe/meta.yaml

```
@@ -84,10 +84,12 @@ outputs:
         - python >=3.9
         - nglview
         - notebook
+        - openff-nagl
+        - openff-nagl-models
```

```suggestion
        - openff-nagl >=0.3.1
```
This brings in `openff-nagl-models` as of this version

Also small preference for a lower constraint here, just in case an old version might be pulled down